spec/controllers/boards_controller_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+RSpec.describe BoardsController, type: :controller do
+
+  before do
+    ApplicationController.any_instance.stub(:current_user).and_return(true)
```

Comment:
stary syntax ..
allow_any_instance_of(ApplicationController).to receive(:current_user) { tutaj nie true tylko user... :P }
ale fajnie jest dodac sobie taka metode do support:
```
# frozen_string_literal: true
module ControllerHelpers
  def sign_in(user = nil)
    user ||= FactoryGirl.build(:user, id: 1)
    allow(controller).to receive(:current_admin_user).and_return(user)
  end
end # module ControllerHelpers

```
i wtedy w kontrolerze poprostu robisz
before { sign_in }